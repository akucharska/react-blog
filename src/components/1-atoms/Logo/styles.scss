.Logo {
  font: {
    size: 4rem;
  }
}
