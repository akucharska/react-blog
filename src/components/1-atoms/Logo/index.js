import React from 'react';
import CSSModules from 'react-css-modules';
import styles from './styles.scss';

function Logo() {
  const styleName='Logo';
  return (
    <div styleName={styleName}>
      React Blog
    </div>
  );
}


export default CSSModules(Logo, styles);
