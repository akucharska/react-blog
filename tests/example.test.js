
import React from 'react';
import renderer from 'react-test-renderer';

import Logo from '../src/components/1-atoms/Logo';

test('example of snapshot', () => {
  const tree = renderer.create(
    <Logo />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
