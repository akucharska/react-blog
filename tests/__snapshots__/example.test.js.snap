// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example of snapshot 1`] = `
<div
  className="Logo"
>
  React Blog
</div>
`;
